ui/lib/fusor_ui/engine.rb

```diff
@@ -34,11 +34,9 @@ class Engine < ::Rails::Engine
       SETTINGS[:fusor_ui] = {:assets => {}} if SETTINGS[:fusor_ui].nil?
 
       SETTINGS[:fusor_ui][:assets][:precompile] = [
-        'fusor_ui/fusor-ember-cli.css',
-        'fusor_ui/vendor-fusor.css',
-        'fusor_ui/bootstrap.css.map',
-        'fusor_ui/fusor-ember-cli.js',
-        'fusor_ui/vendor-fusor.js'
+        'ui/fusor-ember-cli.css',
+        'ui/fusor-ember-cli.js',
+        'ui/vendor.js'
```

Comment:
here was the compilation issue. 2 causes.
1) the path is now ui/ and not fusor_ui/
2) name is vendor.js rather than vendor-fusor.js. Since we are an isolated engine, I don't need to worry about conflicting with Foreman's vendor.js, so I don't need to rename what ember-cli produces.

also, I don't use the vendor.css from ember-cli (bootstrap, etc) since Satellite already has these assets. 
no need for .map files afaik